Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/DirectionIndicator.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Unity.MLAgentsExamples
+{
+    public class DirectionIndicator : MonoBehaviour
+    {
+        public Transform transformToFollow; //ex: hips or body
+        public Transform targetToLookAt; //target in the scene the indicator will point to
+        public float heightOffset;
+        private Vector3 m_StartingPos;
+        
+        void OnEnable()
+        {
+            m_StartingPos = transform.position;
```

Comment:
Only the `y` component of m_StartingPos is used, You do not need to store a Vector3, a float should do.

Reply:
changed it to float.